### PR TITLE
Faq page

### PR DESCRIPTION
**Pull request by sophieH29**
# **Desktop 1**
faq1

# **Desktop 2**

# **Mobile2**

### CODE REVIEW

**Comment by ozasadnyy** [comment]
Can we make it in one line?
image

File: src/pages/faq-page.html [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+        <style is="custom-style" include="shared-styles flex flex-alignment"></style>
+
+        <style>
+             :host {
+                display: block;
+            }
+
+            .content-wrapper {
+                background-color: #4527A0;
+                background-image: linear-gradient(to right, #7E57C2, #4527A0);
```
Comment (ozasadnyy) [inline src/pages/faq-page.html]: should be generic as you config all colors in `shared-styles.html`. 
Have you tried a gray color instead?

File: src/pages/faq-page.html [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+            }
+        </style>
+
+        <div class="content-wrapper">
+            <div class="container">
+                <div class="content" layout justified horizontal wrap></div>
+            </div>
+        </div>
+        <div class="container">
+            <div class="markdown-text">
```
Comment (ozasadnyy) [inline src/pages/faq-page.html]: just add class to `<marked-element>`

File: src/pages/faq-page.html [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+                        colHeader.textContent = head2Element.textContent;
+                        contentDom.node.appendChild(colHeader);
+                        lastHeadElement = colHeader;
+                    }
+
+                    const colContent = document.createElement("a");
+                    colContent.className = 'col-content';
+                    colContent.href = `#${head.id}`;
+                    colContent.textContent = head.textContent;
+                    colContent.addEventListener('click', this._scrollToFaqChapter.bind(this));
```
Comment (ozasadnyy) [inline src/pages/faq-page.html]: try to `bind(this)` in the constructor

File: src/pages/faq-page.html (hunk 1 above)
Comment (sophieH29) [inline src/pages/faq-page.html]: I tried but liked this color the most. Why generic? it's would be used only for this page

File: cors.json [hunk 4]
```diff
@@ -0,0 +1,7 @@
+[
+  {
+    "origin": ["http://localhost:3000", "https://hoverboard-v2-dev.firebaseapp.com/"],
```
Comment (sophieH29) [inline cors.json]: Do you thinks we need to have this in config? It's would be used only once, when you have to run the commantd in gsutil mannually. Moreover this data is static, and does not processed by gulp. And I don't see any big reason that we have to put it there

Comment (ozasadnyy) [inline cors.json]: yes, ideally users should be able to have a functional website after modifying `hoverboard.config.json`

File: src/pages/faq-page.html [hunk 5]
```diff
@@ -13,8 +13,7 @@
             }
 
             .content-wrapper {
-                background-color: #4527A0;
-                background-image: linear-gradient(to right, #7E57C2, #4527A0);
+                background-color: var(--secondary-background-color);
                 width: 100%;
                 overflow: hidden;
                 transition: opacity, opacity 0.5s linear;
```
Comment (ozasadnyy) [inline src/pages/faq-page.html]:
```
transition: opacity var(--animation), opacity var(--animation);
```

File: src/pages/faq-page.html [hunk 6]
```diff
@@ -56,8 +53,20 @@
 
                 [slot="markdown-html"] h2 {
                     font-size: 40px;
-                    max-width: 250px;
+                    max-width: 40%;
```
Comment (ozasadnyy) [inline src/pages/faq-page.html]: do we need this?

File: cors.json [hunk 7]
```diff
@@ -1,6 +1,6 @@
 [
   {
-    "origin": ["http://localhost:3000", "https://hoverboard-v2-dev.firebaseapp.com/"],
+    "origin": ["http://localhost:3000", "https://hoverboard-v2-dev.firebaseapp.com/", "http://localhost:8081", "https://hoverboard-experimental.firebaseapp.com"],
```
Comment (ozasadnyy) [inline cors.json]: please, move it to configs :)

**Pull request merged**